### PR TITLE
Add session route to auth service

### PR DESCRIPTION
**Pull request by megsdevs**

### CODE REVIEW

File: proto/valorem/trade/v1/auth.proto [hunk 1]
```diff
@@ -28,4 +34,10 @@ message NonceText {
 /** Verify request message containing an JSON encoded string of the `SignedMessage` structure (src/auth). */
 message VerifyText {
   string body = 1;
+}
+
+/** The object representing a valid/authenticated sid */
+message SiweSession {
+  H160 address = 1;
+  H256 chain_id = 2;
```
Comment (0xAlcibiades) [inline proto/valorem/trade/v1/auth.proto:44]: Does this need to be an h256? Seems like a u64 would do just fine.

**Review by 0xAlcibiades — CHANGES_REQUESTED** [review]

**Comment by 0xAlcibiades** [comment]
Also, you could add API docs for this to the readme while here for these new endpoints.

File: proto/valorem/trade/v1/auth.proto (hunk 1 above)
Comment (megsdevs) [reply proto/valorem/trade/v1/auth.proto:44]: i copied your spec in the ticket, but yes agree, never bigger than u64

Comment (megsdevs) [reply proto/valorem/trade/v1/auth.proto:44]: will update

**Comment by 0xAlcibiades** [comment]
// The EIP-155 chain_id for the chain for which the quote is requested.
    optional H256 chain_id = 8;
    
    Just going to make this match that.

**Pull request merged**